includes/class-wcpdf-admin.php

```diff
@@ -1300,47 +1300,57 @@ public function handle_heartbeat_received( array $response, array $data ): array
 		return $response;
 	}
 
+	/**
+     * Updates documents data in the "PDF document data" meta box if the generation in the background is finished.
+     *
+	 * @return void
+	 */
 	public function ajax_fetch_pdf_document_data(): void {
-		if ( isset( $_REQUEST['nonce'] ) && wp_verify_nonce( $_REQUEST['nonce'], 'generate_wpo_wcpdf' ) ) {
+		if ( ! isset( $_REQUEST['nonce'] ) || wp_verify_nonce( $_REQUEST['nonce'], 'generate_wpo_wcpdf' ) ) {
```

Comment:
I think should be `security` instead of `nonce` and `! wp_verify_nonce`?

Reply:
Good catch. Thanks.